tests/test_core.py

```diff
@@ -4468,6 +4468,8 @@ def clean(out, err):
 
   def test_mount(self):
     self.set_setting('FORCE_FILESYSTEM', 1)
+    self.set_setting('INCLUDE_FULL_LIBRARY', 1) # uses constants from ERRNO_CODES
+    self.set_setting('ERROR_ON_UNDEFINED_SYMBOLS', 0) # avoid errors when linking in full library
```

Comment:
This seems like a regression, no?

Reply:
The issue here is that the test depends on `ERRNO_CODES` - which is an internal API. So in a sense, FS not automatically including it is a regression and users might notice it. But I think the risk is worth it for emitting smaller code.

Reply:
This seems a little gross.. but I'll take your word for it that there are no obvious better ways to achieve this or to write this test.

Reply:
Another option is to hardcode the constants in the test. I pushed a commit with that now.